### PR TITLE
Removed sqs helm values from aws opta

### PR DESCRIPTION
**Pull request by evalsocket**
fix: https://github.com/flyteorg/flyte/issues/2116

### CODE REVIEW

File: opta/aws/flyte.yaml [hunk 1]
```diff
@@ -88,41 +88,9 @@ modules:
...
-              targetName: "${{module.schedulesQueue.queue_arn}}"
-              scheduleNamePrefix: flyte
-            workflowExecutor:
-              scheme: aws
-              region: "{vars.region}"
-              scheduleQueueName: "${{module.schedulesQueue.queue_name}}"
-              accountId: "{vars.account_id}"
-              reconnectAttempts: 10
-              reconnectDelaySeconds: 30
+        enabled: false
```
Comment (EngHabu) [inline opta/aws/flyte.yaml:91]: We can use native scheduler, right?

Comment (evalsocket) [reply opta/aws/flyte.yaml:91]: yes

File: opta/aws/flyte.yaml [hunk 2]
```diff
@@ -89,40 +89,8 @@ modules:
             "eks.amazonaws.com/role-arn": "${{module.adminflyterole.role_arn}}"
       workflow_scheduler:
         enabled: true
```
Comment (EngHabu) [inline opta/aws/flyte.yaml:91]: Is the native scheduler the default value?

**Review by EngHabu — APPROVED** [review]

File: opta/aws/flyte.yaml (hunk 2 above)
Comment (evalsocket) [reply opta/aws/flyte.yaml:91]: yes

Comment (evalsocket) [reply opta/aws/flyte.yaml:91]: It is coming from eks values https://github.com/flyteorg/flyte/blob/master/charts/flyte-core/values-eks.yaml#L329

**Review by EngHabu — APPROVED** [review]

**Pull request merged**